text/0001-telemetry-without-manual-instrumentation.md

```diff
@@ -56,23 +53,24 @@ There is also a school of thought that we should only be focusing on the bits an
 
 ## Proposal
 
-### What is our desired end state for OpenTelemetry end-users?
+### What is our desired end state for OpenTelemetry end-users
 
 To reiterate much of the above:
+
 * First and foremost, **portable OpenTelemetry instrumentation can be installed without manual source code modification**
 * There’s one “clear winner” when it comes to portable, automatic instrumentation; just like with OpenTracing and OpenCensus, this is a situation where choice is not necessarily a good thing. End-users who wish to contribute instrumentation plugins should not have their enthusiasm and generosity diluted across competing projects.
 * As much as such a thing is possible, consistency across languages
 * Broad coverage / “plugin support”
 * Broad vendor support for OpenTelemetry
 * All other things being equal, get all of these ^^ benefits ASAP!
 
-### What's the basic proposal?
+### What's the basic proposal
```

Comment:
This is a content change. I don't think this belongs here.

Reply:
There is a rule in the markdown lint to not use punctuation signs in headers.

Reply:
And I would argue that this change makes any significant "content change" :)

Reply:
Since this is used rather commonly throughout a repo and in my opinion shouldn't be taken as a mistake we could consider disabling this rule.

Reply:
Change config to allow `?` in the headers. I think not allowing `.` is still valuable.